lib/RocketMap_MAD.php

```diff
@@ -623,6 +655,12 @@ public function generated_exclude_list($type)
             foreach ($pokestops as $pokestop) {
                 $data[] = $pokestop['incident_grunt_type'];
             }
+	} elseif ($type === 'raidbosslist') {
+            $gyms = $db->query("SELECT distinct pokemon_id FROM raid WHERE pokemon_id > 0 AND end > UTC_TIMESTAMP() order by pokemon_id;")->fetchAll(\PDO::FETCH_ASSOC);
+            $data = array();
+            foreach ($gyms as $gym) {
+                $data[] = $gym['raid_pokemon_id'];
```

Comment:
Change this to
```php
$data[] = $gym['pokemon_id']
```

After that everything else is working on my instance

Reply:
check